src/amp-tags.js

```diff
@@ -9,7 +9,7 @@ function storyPageAmpTags(story) {
 }
 
 export function StoryAmpTags(seoConfig, config, pageType, data, opts) {
-  if(pageType == 'story-page' && seoConfig.ampStoryPages && get(data, ["data", "story", "is-amp-supported"]))
+  if(pageType == 'story-page' && get(data, ["data", "story", "is-amp-supported"]))
```

Comment:
did we forget to remove this when we implemented `is-amp-supported` feature?

Reply:
i suppose so.